static/app/components/sidebar/index.tsx

```diff
@@ -621,6 +622,7 @@ function Sidebar() {
       icon={<IconSearch />}
       label={<GuideAnchor target="explore">{t('Explore')}</GuideAnchor>}
       id="explore"
+      isNew
```

Comment:
```suggestion
```

confusingly, the "Explore" accordion is "not" new!

Reply:
Good catch, i thought it was, because it's tied to the same `performance-insights` flag